src/expm.jl

```diff
@@ -0,0 +1,51 @@
+@inline expm(A::StaticMatrix) = _expm(Size(A), A)
+
+@inline function _expm(::Size{(1,1)}, A::StaticMatrix)
+    T = promote_type(typeof(exp(zero(eltype(A)))), Float32)
+    newtype = similar_type(A,T)
+
+    (newtype)((exp(A[1]), ))
+end
+
+@inline function _expm{S<:Real}(::Size{(2,2)}, A::StaticMatrix{S})
```

Comment:
I think there is a branch somewhere where we can merge backports. It's just more work.